language-server/protocol/extProtocol.ml

```diff
@@ -284,6 +301,9 @@ module Request = struct
     | "vscoq/documentState" ->
       let+ params = Lsp.Import.Json.message_params params DocumentStateParams.t_of_yojson in
       Pack (DocumentState params)
+    | "vscoq/documentProofs" ->
+      let+ params = Lsp.Import.Json.message_params params DocumentProofsParams.t_of_yojson in
+      Pack (DocumentProofs params)
```

Comment:
I'm wondering if we can easily version these custom queries, eg `"vscoq/documentProofs/v1"` so that we can update the format later without necessarily breaking badly the plugins that use this extension

Reply:
So after thinking about it, we make it accessible through a function in the client. If we need new versions it should be easy to keep everything wired on our end when plugins use our extension.